lib/user.dart

```diff
@@ -155,7 +155,7 @@ class CurrentUser {
   }
 
   String getFullName() {
-    return _name ?? "Biletskiy,Yevgen";
```

Comment:
This should be removed.

Reply:
the -,- not the already removed line

Reply:
so you think it should return a String? instead of a String 
?

Reply:
I appreciate this. However, I don't fully understand what you mean. I will call you in a few.

Reply:
This is fine.